src/Expensify.js

```diff
@@ -131,8 +134,26 @@ class Expensify extends PureComponent {
         ActiveClientManager.init();
     }
 
+    showSplash() {
+        Log.info('[BootSplash] showing splash screen', false);
+        BootSplash.show({fade: true});
+    }
+
     hideSplash() {
-        BootSplash.hide({fade: true});
+        Log.info('[BootSplash] hiding splash screen', false);
+        BootSplash.hide({fade: true})
+            .catch(error => Log.alert('[BootSplash] hiding failed', {message: error.message, error}, false));
+    }
+
+    reportBootSplashStatus() {
+        BootSplash.getVisibilityStatus()
+            .then((status) => {
+                Log.info('[BootSplash] Splash screen status', false, {status});
+
+                if (status === 'visible') {
+                    Log.alert('[BootSplash] Still visible. Current <Expensify /> props', {props: _.omit(this.props, 'children')}, false);
```

Comment:
It might be a good idea to make this `Log.hmmm` so we don't end up spamming the logs if there are a lot of instances of this. 

Additionally, I still think `splash screen is still visible` is better than `Still visible.`, and `Current <Expensify /> props` is now unnecessary.

Reply:
Also, I'm not sure it super matters, since the authTokens are encrypted, but it's probably best not to log the authTokens from the props. Instead, we could just log whether or not an authToken exists.